app.test.js

```diff
@@ -51,14 +51,15 @@ describe("GET/api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-        expect(body.article).toHaveProperty("author")
-        expect(body.article).toHaveProperty("title")
-        expect(body.article).toHaveProperty("article_id")
-        expect(body.article).toHaveProperty("body")
-        expect(body.article).toHaveProperty("topic")
-        expect(body.article).toHaveProperty("created_at")
-        expect(body.article).toHaveProperty("votes")
-        expect(body.article).toHaveProperty("article_img_url")
+        expect(body.article).toHaveProperty("author", expect.any(String))
+        expect(body.article).toHaveProperty("title", expect.any(String))
+        expect(body.article).toHaveProperty("article_id", expect.any(Number))
+        expect(body.article).toHaveProperty("body", expect.any(String))
+        expect(body.article).toHaveProperty("topic", expect.any(String))
+        expect(body.article).toHaveProperty("created_at", expect.any(String))
+        expect(body.article).toHaveProperty("votes", expect.any(Number))
+        expect(body.article).toHaveProperty("article_img_url", expect.any(String))
+        expect(body.article).toHaveProperty("comment_count", expect.any(String))
```

Comment:
Instaead of editing this test, this task should really have a new test. This original test should not fail due to this feature being implemented. Have a go at writing a new test, and restoring this test to how it was.